test/extended/operators/cluster.go

```diff
@@ -136,10 +137,14 @@ func crashloopingContainerCheck(podFilters ...podFilter) {
 			e2e.Logf("Pod status %s/%s ignored for being pending", pod.Namespace, pod.Name)
 			continue
 		}
+		if pod.Status.StartTime.After(testStartTime) {
```

Comment:
This check should not be present.  A pod that was started before this test began (which is random in time to start) should definitely have time to complete its lifecycle if it is pending.